docs/standards.md

```diff
@@ -0,0 +1,167 @@
+## SG2 Standards Checklist
+
+Since SG2 is a living design library that is home to reusable patterns, it's important for all contributing developers to attempt to create patterns the same way. 
+
+Any new work contributed into SG2 must meet the following standards:
+
+### PatternLab Organization
+
+#### How patterns are organized:
+- Patterns should be listed alphabetically in the pattern list. No numbers (e.g. `atoms/forms/text-input` instead of `00-atoms/10-forms/05-text`). 
+- Talk about where SCSS and JS files should go (in a separate .scss directory?), as well as Twig and .md files (no subdirectories?). We may still need to make a decision about this.
+
+##### Atoms
+- An atom should be the basic HTML elements.
+- They can sometimes be more than one element (e.g. an unordered list).
+
+##### Molecules
+- A molecule should include most of the markup and pull in atoms as necessary.
+- Not every element in a molecule needs to be an atom.
+
+##### Organisms
+- Organisms should be very little markup, it should be mostly wrappers around molecules or atoms.
+
+##### Templates
+- Layouts should be used for templates but not molecules
+
+##### Pages
+Add a description here.
+
+##### Base
+Add a description of the "base" patterns here. Do they always get reused, or no?
+
+Contains:
+- "Colors" atom showing variables and base color palette.
+
+##### Meta
+Add a description of the "meta" patterns here. This section still needs to be added to SG2.
+
+### How to classify/name patterns
+All pattern class names should follow standard [BEM practices](http://getbem.com/naming/) and be prefixed according to their theme (e.g. `ama__`). 
+
+Below is an example of how a breadcrumb organism for the default AMA theme may appear:
+```html
+<section class="ama__breadcrumbs ">
+  <nav class="ama__breadcrumbs__container">
+    <ol class="ama__breadcrumbs__items">
+      <li class="ama__breadcrumbs__item"><a href="#">Link 1</a></li>
+      <li class="ama__breadcrumbs__item"><a href="#">Link 2</a></li>
+    </ol>
+  </nav>
+</section>
+```
+This pattern shows that `breadcrumbs` is a block of the `ama` theme and has the elements `container`, `items` and `item`.
+
+#### Pattern classification
+See "PatternLab Organization" above. 
```

Comment:
@chelsiejohnston what questions do you want this section to answer to potential developers?

Reply:
"I'm making a thing. How do I figure out how to break it up into atoms/molecules/organisms/templates/pages so that it fits in with the SG?"